app/javascript/components/student/TracksList.tsx

```diff
@@ -29,6 +29,7 @@ export default ({
   tagOptions: readonly TagOption[]
   request: Request
 }): JSX.Element => {
+  const [staleTime, setStaleTime] = useState(1000)
```

Comment:
Legacy?

Reply:
Yes

Reply:
But this isn't used?

Reply:
No. It's not needed (and not used in the code as far as I can tell).

Reply:
I used this to try out a concept, and forgot to delete.